sox.css

```diff
@@ -447,6 +447,10 @@
     text-align: center;
 }
 
+.main {
```

Comment:
Would you mind naming this similar to other classes/IDs, so that it has the feature's name in it?

Something along the lines of `.sox-flagPercentProgressBar-container` would be great! :)

Reply:
It must be OK, now.

Reply:
Thanks!